Builds/VisualStudio/generate_rust_bridge.bat

```diff
@@ -10,13 +10,13 @@ set "final_source=%out_dir%\RustBridge.cpp"
 
 mkdir "%out_dir%\temp" >nul 2>nul
 
-%1 ..\..\src\rust\src\lib.rs --cfg test=false --header --output "%temp_header%"
+%1 ..\..\src\rust\src\bridge.rs --cfg test=false --header --output "%temp_header%"
```

Comment:
Just double checking, is this an intentional change for a broken windows build in master?

Reply:
Yeah, this is intentional because of the broken Win build (this has been just changed in the makefile).